openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -533,7 +540,6 @@ OpenAssessment.ResponseView.prototype = {
         var fileType = null;
         var fileName = '';
         var errorCheckerTriggered = false;
-        var sel = $('.step--response', this.element);
```

Comment:
I have removed this and instead used `$(this.element).find('. file__upload')` to find the file upload button. The reason is, In Jasmine tests `.step--response` element refers to `this.element` . Hence the selecot `$('.step--response', this.element)` returns 0 desendets  because  the selector is restricted to match only descendants of a determined context (i.e. `this.element`)